### PR TITLE
Clean up documentation for 1.2

### PR DESCRIPTION
**Pull request by khaeru**
This fixes warnings and errors in the documentation build which were noticed during other prep for 1.2.

Preview: https://message.iiasa.ac.at/en/rc1.2-doc-cleanup/

### CODE REVIEW

**Comment by gidden** [comment]
lgtm, will wait for CI

**Comment by khaeru** [comment]
CI passes, but will wait 'til iiasa/ixmp#150 and #206 are in to see if the cross-repo inclusion needs cleanup.

**Comment by gidden** [comment]
#206 is still blocking right?

File: message_ix/model/MESSAGE/sets_maps_def.gms [hunk 1]
```diff
@@ -202,9 +202,13 @@ Alias(mode, mode2);
 *      - :math:`\widehat{t} \in \widehat{T}^{LAND} \subseteq \widehat{T}`
 *      - Mapping set of technology types and land use
 *    * - balance_equality (commodity,level)
-*      - :math:`c \in C^{FULL} \subseteq C`
-*      - :math:`l \in L^{FULL} \subseteq L`
-*      - Commodities and level related to `Equation COMMODITY_BALANCE_LT`_
+*      - .. math::
+*
+*           c \in C^{FULL} \subseteq C
```
Comment (danielhuppmann) [inline message_ix/model/MESSAGE/sets_maps_def.gms]: not sure why we used `C^{FULL}` here in the first place - maybe just write `c, l \in C, L`?

Comment (khaeru) [reply message_ix/model/MESSAGE/sets_maps_def.gms]: Not sure either! I was just cleaning up the extra bullet item, which was breaking the list-to-table conversion.

Will make that change.

**Comment by khaeru** [comment]
I cleaned up the docs in #206, so it no longer blocks this. Will merge once tests past.

**Pull request merged**